.github/workflows/ci-deptycheck.yml

```diff
@@ -119,12 +131,13 @@ jobs:
           name: built-deptycheck
           path: ${{ env.pack_dir_file }}
 
-  #######$###################
+  ###########################
   # Build DepTyCheck's docs #
-  ########$##################
+  ###########################
 
   deptycheck-build-docs:
     name: Build the docs
+    needs: validate_trigger
```

Comment:
We need to think if this can be made simpler, but this is definitely okay for now